layouts/partials/hooks/body-end.html

```diff
@@ -2,3 +2,48 @@
   {{ partial "script.html" (dict "src" "js/tracing.js") -}}
 {{ end -}}
 {{ partial "script.html" (dict "src" "js/navScroll.js") -}}
+
```

Comment:
Move all of this extra content into it's own partial, say `layouts/partials/scroll-to-top.html`.

As for the JS, and I think I mentioned this before, do the following:

- Move the JS into its own file, say, `assets/js/scrollToTop.js`
- Use `layouts/partials/script.html` to bring in the script content.

Reply:
But @svrnm said i handle this differently already

Reply:
But @svrnm said i handle this differently already

Reply:
Yeah all he just suggested was what i did initially already, i had the scrollToTop.js file in the js folder but you suggested i implement the javascript code on the body-end.html file that way getting rid of the scrollToTop.js file seeing as it was no longer needed, but i am confused now @svrnm

Reply:
My apologies @svrnm @chalin i took my time to re-read the whole conversation and realized i got it mixed up, totally understand now, sorry for the misunderstanding (i have been feverish the past few days). Implementing the changes right now. Thanks so much.

Reply:
@bintus-ux no worries, everyone of us is misreading things from time to time and from own experience I know that beign a non-native english speaker adds an additional source of potential misunderstanding, that's why it is always good to ask more questions! 

I hope you feel better now!